src/group_cmd.cpp

```diff
@@ -480,11 +480,11 @@ CommandCost CmdAlterGroup(DoCommandFlags flags, AlterGroupMode mode, GroupID gro
 
 			GroupStatistics::UpdateAutoreplace(g->owner);
 
-			if (!HasBit(g->livery.in_use, 0) || !HasBit(g->livery.in_use, 1)) {
+			if (!g->livery.in_use.All({Livery::Flag::Primary, Livery::Flag::Secondary})) {
```

Comment:
Doesn't this change the meaning of the condition to check if both flags are false, and not if either flag is false? Or am I misunderstanding the All check?

Reply:
De Morgan's Theorem.

`!x || !y` is equivalent to `!(x && y)`